plugins/ocm-backend/src/helpers/config.ts

```diff
@@ -97,3 +98,7 @@ export const readOcmConfigs = (config: Config): OcmConfig[] => {
     .keys()
     .map(id => getHubClusterFromConfig(id, ocmConfigs.getConfig(id), config));
 };
+
+export const getOwnerFromHub = (hubConfig: Config): string => {
+  return hubConfig.getOptionalString(OWNER_KEY) || 'unknown';
+};
```

Comment:
I think adding some tests is a good idea. As you can see in my other comment below, I think we need those tests here. 😄

Reply:
I've added tests in #228 together with a fix to the `url` validation since it wasn't working properly. Once it is merged I'll update the tests to also check for the owner configuration.